_sass/modules/_m-search.scss

```diff
@@ -35,7 +35,14 @@
       margin: rem(80px) auto;
     }
 
-    @media screen and (min-width: $screen-m) {
+    @media screen and (min-width: 768px) {
+      display: block;
+      padding-left: rem(95px);
+      padding-right: rem(95px);
+      max-width: rem(350px);
```

Comment:
You don't need to add paddings neither max-width. Those values should be at the generic class. 
You can check this as an example: https://github.com/Vizzuality/goal16/blob/develop/_sass/modules/_m-aside-compare.scss#L43

Change the media query to match $screen-s. It is very weird to have directives for only one pixel, don't you think?